docs/api.md

```diff
@@ -16,6 +16,7 @@ It supports these props, while passing any others through to the `children`:
 | noInline      | `PropTypes.bool`   | Doesn’t evaluate and mount the inline code (Default: `false`). Note: when using `noInline` whatever code you write must be a single expression (function, class component or some `jsx`) that can be returned immediately. If you'd like to render multiple components, use `noInline={true}` |
 | transformCode | `PropTypes.func`   | Accepts and returns the code to be transpiled, affording an opportunity to first transform it                                                                                                                                                                                                 |
 | language      | `PropTypes.string` | What language you're writing for correct syntax highlighting. (Default: `jsx`)                                                                                                                                                                                                                |
+| enableTypeScript      | `PropTypes.bool` | Enables TypeScript support in transpilation. (Default: `true`)                                                                                                                                                                                                                |
```

Comment:
I think this was mentioned in the issues over time, and I confess I don't remember any of the details, but instead of doing a bool flag here and wrapping a different optioned sucrase `transform` would we want to consider just taking sucrase transform options directly and giving some suggestions in documentation here?